test-curves/src/bls12_381/g1.rs

```diff
@@ -52,6 +53,77 @@ impl short_weierstrass::SWCurveConfig for Parameters {
     }
 }
 
+impl WBParams for Parameters {
```

Comment:
Are these constants generated by a sage script or something? would be good to have a reference there